### PR TITLE
Better socmed images

### PR DESCRIPTION
**Pull request by benabraham**
New image and all the meta tags Facebook and Twitter want (some not necessarily, but it’s better to have them).

Facebook requires `og:image` content to be an **absolute URL**. The only solution I was able to do is with hardcoded protocol. Feel free to find a better one.

### CODE REVIEW

File: pyconcz_2017/templates/_base.html [hunk 1]
```diff
@@ -9,7 +9,17 @@
 
   <title>{% block meta_title %}PyCon CZ 2017{% endblock %}</title>
   {% block meta_head %}
-    <meta property="og:image" content="{% static "img/pyconlogo.png" %}">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="PyCon CZ 2017">
+    <meta property="og:site_name" content="PyCon CZ 2017">
+    <meta property="og:description" content="PyCon CZ 2017 aims to be the largest annual gathering for the Python community in the Czech Republic.">
+    <meta property="og:image" content="https://{{ request.get_host }}{% static "img/social-media-image.png" %}">
```
Comment (benabraham) [inline pyconcz_2017/templates/_base.html:16]: line 16: Unfortunately `{{ request.scheme }}` instead of `https` didn’t seem to work.

**Comment by benabraham** [comment]
@starenka tl;dr: I need to get absolute url for one static media file, any ideas better than my hardcoded one?

**Comment by benabraham** [comment]
Halp plz? Anybody with 1337 Django skills? 
@starenka @HonzaKral @honzajavorek @tricoder42 @Luminaar 

Motivation: 
1) This will fix suboptimal images when we share any page of Pycon.cz on social media sites. 📈 
2) No more notifications from me. 🛑

**Comment by tricoder42** [comment]
I don't mind hardcoded `https`. Everything is secure by default and website should *never* be served from plain `http`.

If you really want it, try `request.is_secure`:

```html
<meta property="og:image" content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.get_host }}{% static "img/social-media-image.png" %}">
```

https://docs.djangoproject.com/en/1.11/ref/request-response/#django.http.HttpRequest.is_secure

**EDIT:** Sorry, `request.is_secure` actually uses `request.scheme` under the hood. So it's probably better to use `scheme` directly, but if it doesn't work... just merge it with hardcoded scheme ;)

File: pyconcz_2017/templates/_base.html (hunk 1 above)
Comment (honzakral) [reply pyconcz_2017/templates/_base.html:16]: https://en.wikipedia.org/wiki/URL#prurl

**Comment by honzakral** [comment]
https://en.wikipedia.org/wiki/URL#prurl should work

**Comment by benabraham** [comment]
@HonzaKral I'm pretty (not 100 %) sure I tried that and it didn't work.

I think we can keep it simple (better than complex. :) and leave hardcoded protocol there.

In the case somebody ran the site on http for whatever reason, it'd be up to them to fix it.

**Comment by honzajavorek** [comment]
Cool! 👍

**Pull request merged**